tests/header.rs

```diff
@@ -13,6 +13,9 @@ fn minimal_hdr() {
         datatype: 2,
         bitpix: 8,
         pixdim: [0., 3., 3., 3., 0., 0., 0., 0.],
+        srow_x: [0.; 4],
+        srow_y: [0.; 4],
+        srow_z: [0.; 4],
         vox_offset: 0.,
         scl_slope: 0.,
         scl_inter: 0.,
```

Comment:
You might want to create a `get_test_header` or something for those 15-lines header creations. At least the first three are identical.

Reply:
Indeed, I will create helper functions for building the headers for "minimal" test files.